### PR TITLE
Add public_relations role

### LINKED ISSUE #201: Allow 'public_relations' role to access archived pins

### PR DESCRIPTION
**Pull request by supasate**
Fix #201 

Note: Currently, we allow all roles to GET all pins. We might have to re-check this later.

### CODE REVIEW

File: test/fixtures/public_relations_user.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const DEPARTMENT_PUBLIC_RELATIONS_ID = require('./constants').DEPARTMENT_PUBLIC_RELATIONS_ID;
+const USER_PUBLIC_RELATIONS_ID = require('./constants').USER_PUBLIC_RELATIONS_ID;
+// roles
+const PUBLIC_RELATIONS = require('../../src/constants/roles').PUBLIC_RELATIONS;
+
+module.exports = {
+  _id: USER_PUBLIC_RELATIONS_ID,
+  name: 'YouPin Super Admin',
```
Comment (parnurzeal) [inline test/fixtures/public_relations_user.js]: let's use other name.

File: test/fixtures/public_relations_user.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  phone: '081-985-2586',
+  fb_id: 'youpin_fb_id',
+  // hash of 'youpin_admin' password
+  password: '$2a$10$iorOMFOPboPeF20W20DKruey2UXXa4eOQSuReOMlxXnqNe5t6Egaq',
```
Comment (parnurzeal) [inline test/fixtures/public_relations_user.js]: let's use password 'youpin_xxx' 
how about 'youpin_public_relations'?

**Comment by parnurzeal** [comment]
LGTM. After fix, let me know again.

File: test/fixtures/public_relations_user.js (hunk 2 above)
Comment (supasate) [reply test/fixtures/public_relations_user.js]: good. will fix.

File: test/fixtures/public_relations_user.js (hunk 1 above)
Comment (supasate) [reply test/fixtures/public_relations_user.js]: Okay. Might be `YouPin Public Relations`.

**Comment by supasate** [comment]
@parnurzeal Fixed!

**Pull request merged**